src/Solnet.Rpc/SolanaRpcClient.cs

```diff
@@ -27,11 +27,13 @@ public class SolanaRpcClient : JsonRpcClient, IRpcClient
         /// <param name="url">The url of the node exposing the JSON RPC API.</param>
```

Comment:
is it just me or is rider fight VS in regards to indentation and line wrapping?

Reply:
Very possible